pkg/runtime/runtime.go

```diff
@@ -52,3 +62,27 @@ func Lookup(path string) (Interface, bool) {
 	r, ok := runtimes[ext]
 	return r, ok
 }
+
+const (
+	// Separator converted to string to abort pathof at root.
+	sep = string(filepath.Separator)
+)
+
+// Pathof attempts to find the path of the given filename.
+//
+// The method recursively descends the path until the given
```

Comment:
"descends" might be confusing

```suggestion
// The method recursively visits parent dirs until the given
```